coordinator/src/main/scala/filodb.coordinator/queryplanner/SingleClusterPlanner.scala

```diff
@@ -894,4 +952,31 @@ class SingleClusterPlanner(val dataset: Dataset,
     }
     PlanResult(metaExec)
   }
+
+  /**
+   * Materialize an Aggregate without the pushdown optimization.
+   * @param forceDispatcher If occupied, forces this Aggregate to be materialized with the dispatcher.
+   *                        Only this root plan has its dispatcher forced; children are unaffected.
+   *                        ####### The dispatcher is applied regardless of forceInProcess #######
+   */
+  private def materializeAggregateNoPushdown(qContext: QueryContext,
```

Comment:
Basically a copy-paste of the default planner's implementation, but with both `forceDispatcher` and  `forceInProcess`.